### ISSUE TITLE
Bootloader is non-functional

### ISSUE THREAD

**Issue by WillB97** [srobo/power-v4-fw#15]
The current bootloader this generates doesn't establish a USB connection. This may be an issue in srobo/dfu-bootloader

**Comment by RealOrangeOne** [comment]
Does this mean we may need to reflash all the boards in the kits we recently packed?

**Comment by WillB97** [comment]
No, we only changed the firmware not the the bootloader. This is simply that we cannot build a new bootloader from the current code.
